### ISSUE TITLE
Azcopy -How to skip the files which are in use

### ISSUE THREAD

**Issue by pavithranpv** [Azure/azure-storage-azcopy#1037]
### Which version of the AzCopy was used? 

Azcopy 8
### Which platform are you using? (ex: Windows, Mac, Linux)
Windows
### What command did you run?

AzCopy /Source:$localPath /Dest:$containerSASURI /V:$logfile /S /XO /Y

### What problem was encountered?
I am using azcopy to move files from local disk to Azure blob . In the local disk , there some files in use so the copy fails for these files with the error . Is there a way to skip the files in use while copying ? So that i can avoid getting the below error .

The process cannot access the file '\\?\UNC\xxxxxxxxxxxxxxxx00.sqlaudit' because it is being used by another process..

### How can we reproduce the problem in the simplest way?
In short , i need to skip the open files while doing azcopy from local disk to blob storage

**Comment by zezha-msft** [comment]
Hi @pavithranpv, you could switch to AzCopy V10, which is more fault tolerant and does not fail the job despite some files failing.
